src/Synapse/Synapse/Commands/ManagementCommands/SqlPool/ConvertAzureSynapseVulnerabilityAssessmentScan.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Synapse.Common;
+using Microsoft.Azure.Commands.Synapse.Models;
+using Microsoft.Azure.Commands.Synapse.VulnerabilityAssessment.Model;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Synapse
+{
+    [Cmdlet("Convert", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + SynapseConstants.SynapsePrefix + SynapseConstants.SqlPool + SynapseConstants.VulnerabilityAssessmentScan,
+        SupportsShouldProcess = true)]
+    [OutputType(typeof(PSVulnerabilityAssessmentScanExportModel))]
+    public class ConvertAzureSynapseVulnerabilityAssessmentScan : SynapseManagementCmdletBase
+    {
+        private const string ConvertByVulnerabilityAssessmentScanRecordModelParameterSet = "ConvertByVulnerabilityAssessmentScanRecordModelParameterSet";
+        private const string ConvertByNameParameterSet = "ConvertByNameParameterSet";
+ 
+        [Parameter(Mandatory = false, HelpMessage = HelpMessages.ResourceGroupName)]
```

Comment:
>[Parameter(Mandatory = false, HelpMessage = HelpMessages.ResourceGroupName)] [](start = 8, length = 76)

Do we need to add "ValueFromPipelineByPropertyName = true" for `ResourceGroupName`?